content/en-US/api/USBEndpoint/USBEndpoint.md

```diff
@@ -0,0 +1,73 @@
+---
+recipe: api-feature
+title: 'USBEndpoint'
+mdn_url: /en-US/docs/Web/API/USBEndpoint
+specifications: https://wicg.github.io/webusb/#endpoints
+browser_compatibility: api.USBEndpoint
+---
+
+## Description
+
+The `USBEndpoint` interface of the WebUSB API provides information about an endpoint provided by the USB device. An endpoint represents a unidirectional data stream into or out of the device. Endpoints come in three flavors,
+
+## Constructor
+
+Creates a new `USBEndpoint` object which will be populated with information about the endpoint on the provided `USBAltenateInterface` with the given endpoint number and transfer direction.
+
+## Properties
+
+**`USBEndpoint.endpointNumber`**
+
+Returns this endpoint's "endpoint number" which is a value from 1 to 15 extracted from the `bEndpointAddress` field of the endpoint descriptor defining this endpoint. This value is used to identify the endpoint when calling methods on `USBDevice`.
+
+**`USBEndpoint.direction`**
+
+Returns the direction in which this endpoint transfers data. One of,
+
+* `"in"` - data is transferred from device to host
+* `"out` - data is transferred from host to device
+
+**`USBEndpoint.type`**
+
+Returns the type of this endpoint. One of,
+
+* `"bulk"` - Bulk endpoints provide reliable data transfer for large payloads. Data send through a bulk endpoint is guaranteed to be delivered or generate an error but may be preempted by other data traffic.
```

Comment:
```suggestion
* `"bulk"` - Provides reliable data transfer for large payloads. Data sent through a bulk endpoint is guaranteed to be delivered or generate an error but may be preempted by other data traffic.
```

Reply:
Definition lists are one of the few places writers are allowed to use a sentence fragment. It eliminates duplicate information. Generally you start the fragment with a present tense verb.